frontend/src/metabase-lib/v1/queries/structured/Filter.ts

```diff
@@ -354,7 +354,7 @@ export default class Filter extends MBQLClause {
     return this.set(setFilterOptions(this, options));
   }
 
-  formattedArguments(maxDisplayValues?: number = 1) {
+  formattedArguments(maxDisplayValues: number = 1) {
```

Comment:
How was this related to webpack -> SWC migration?

Reply:
@nemanjaglumac @WiNloSt it's not correct typescript syntax, so swc complained about it 😄

Reply:
Thanks